src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -657,10 +657,10 @@
     <value>Cyan</value>
   </data>
   <data name="SkyBlueColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Sky Blue</value>
+    <value>Sky blue</value>
   </data>
   <data name="DarkGrayColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Dark Gray</value>
+    <value>Dark gray</value>

```

Comment:
Same goes for these, should probably stay capitalized since they're proper nouns for colors.